packages/dictionary/src/dataFileTemplates.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2025 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+import { Json2tsv } from 'json2tsv';
+import { Schema } from './types';
+
+interface CreateDataFileTemplateOptions {
+	delimiter?: string;
+	quote?: string;
+}
+
+export const createDataFileTemplate = (
+	schema: Schema,
+	options?: { delimiter?: string },
+): { fileName: string; content: string } => {
+	const delimiter = options?.delimiter ?? '\t';
+	const fields = schema.fields || [];
+
+	const templateRow = fields.reduce((acc: { [key: string]: string }, field: { name: string }) => {
+		acc[field.name] = '';
+		return acc;
+	}, {});
+
+	const tsvParser = new Json2tsv({ delimiter, quote: '' });
```

Comment:
Looks like you meant to use the `quote` property but didn't. Should we update this? Either remove the option or use it (probably remove it).